homework/index.js

```diff
@@ -29,19 +29,127 @@
     });
     return elem;
   }
-
+  function singlePageApplication(arr) {
+    //header
```

Comment:
If you run `npm run lint` in your terminal you'll see 19 errors, so I'm not gonna point out all the errors again. Please check your `npm run lint` and take a look at the errors, most of them are really easy to solve. Next time check your PR if all checks are passing!